Donut/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
```

Comment:
PEP doesn't really care as long as it's not `from module import *`. It's really up to us, so if `import flask` and `flask.function()` seems cleaner let's go with that.